pennylane/ops/qubit/parametric_ops.py

```diff
@@ -2306,13 +2306,18 @@ class IsingXX(Operation):
     r"""
     Ising XX coupling gate
 
-    .. math:: XX(\phi) = \begin{bmatrix}
+    .. math:: XX(\phi) = \exp(-i \frac{\theta}{2} (X \otimes X)) =
+        \begin{bmatrix} =
             \cos(\phi / 2) & 0 & 0 & -i \sin(\phi / 2) \\
             0 & \cos(\phi / 2) & -i \sin(\phi / 2) & 0 \\
             0 & -i \sin(\phi / 2) & \cos(\phi / 2) & 0 \\
             -i \sin(\phi / 2) & 0 & 0 & \cos(\phi / 2)
         \end{bmatrix}.
 
+    .. note::
+
+        :math:`XX(0) = I, XX(\pi) = i (X \otimes X)`
+
```

Comment:
Adding a bit of text and itemising the equation for each operation may be worth it (the example is just a suggestion).
```suggestion

        Special cases of using the :math:`XX` operator include:
        
        * :math:`XX(0) = I`;
        * :math:`XX(\pi) = i (X \otimes X)`.

```